templates/navigation-mobile-menu.twig

```diff
@@ -0,0 +1,77 @@
+<div id="navigation-mobile-menu">
+  <button class="nav-menu-close" type="button"
+    aria-label="{{ __( "Close navigation mobile menu", "planet4-master-theme" ) }}"
+    aria-expanded="false"
+    data-bs-toggle="open"
+    data-bs-target="#navigation-mobile-menu">
+    <span class="visually-hidden">
+      {{ __( "Close navigation mobile menu", "planet4-master-theme" ) }}
+    </span>
+  </button>
+  <div class="navigation-mobile-menu-header">
+    <a class="site-logo" href="{{ data_nav_bar.home_url }}">
+      {% include "blocks/site_logo.twig" %}
+    </a>
+		{% if site_languages is not empty %}
+			{% set current_lang = site_languages|filter(i => i.active)|first %}
+			<button class="nav-languages-toggle" type="button"
+				aria-label="{{ __( "Choose language", "planet4-master-theme" ) }}"
+				aria-expanded="false"
+				data-bs-toggle="open"
+				data-bs-target="#nav-languages"
+			>{{ current_lang.code|capitalize }}</button>
+			<nav
+				id="nav-languages"
+				class="nav-languages"
+				aria-label="{{ __( 'Languages of the site', 'planet4-master-theme' ) }}"
+			>
+				<ul>
+				{% for key,item in site_languages %}
+				<li class="nav-item {{ item.active ? "current-language" : "" }}">
+					<a class="nav-link" href="{{ item.url }}">{{ item.native_name }}</a>
+					<span aria-hidden="true">&nbsp;|&nbsp;</span>
+				</li>
+				{% endfor %}
+				</ul>
+			</nav>
+		{% endif %}
+  </div>
+
+	<nav aria-label="{{ __( 'Menu items', 'planet4-master-theme' ) }}">
+		<ul id="accordion-list" class="accordion">
+			<li class="navigation-mobile-menu-item">
+				<a
+					class="nav-link"
+					href="#"
+					data-ga-category="Submenu Navigation"
+					data-ga-action="Home"
+					data-ga-label="{{ page_category }}">
+						{{ __( "Home", "planet4-master-theme" ) }}
+				</a>
+			</li>
+
+			{% if navbar_menu_items is not empty %}
+				{% include "navigation-mobile-menu-items.twig" with { menu: navbar_menu_items } %}
+			{% endif %}
+
+			{% if donate_menu_items is not empty %}
+				{% include "navigation-mobile-menu-items.twig" with { menu: donate_menu_items } %}
+			{% endif %}
+		</ul>
+	</nav>
+
+	{% if donate_menu_items[0] is not null %}
+		{% set donate_menu = donate_menu_items[0] %}
```

Comment:
very subjective opinion :grin: twig has a [`donate_menu_items|first`](https://twig.symfony.com/doc/1.x/filters/first.html) filter, that looks cuter than `[0]`